core/app/components/SubmitButton.tsx

```diff
@@ -82,9 +82,9 @@ export const SubmitButton = ({
 			setButtonState("success");
 		} else if (isSubmitError) {
 			setErrorState();
+		} else {
+			setButtonState("idle");
 		}
-
-		setButtonState("idle");
```

Comment:
not sure how it was in this state for this long, but basically the loading wasn't working at all, it was just stuck in the idle always